docs/dev/cli-commands.md

```diff
@@ -0,0 +1,38 @@
+## CLI Basics
+
+The main tasks of the CLI component are:
+- parse and call the commands
+- call the method to transform a given topology to a custom platform (start / stop / transform platform)
+- show current status of transformation/ deployment
+- print detailed logs if wanted
+- show supported platforms
+- create threads to provide asynchronous operations
+
+## Supported Commands
+
+Each of the following commands except help have these options: `-d` or `--debug`, `-u` or `--usage`, `-v` or `--verbose`.
+Parameters of the commands are represented as `<name>`.
+
+Options explanation:
+- Debug: enable/ disable Debugging
+- Usage: Explain the usage of the specific command
+- Verbose: enable/ disable more detailed Information
+
+| Command | Description | additional Options |
+| ------- | ----------- | ------------------ |
+| add | uploads a specific CSAR Archive to the Transformator, Usage: `add <CSARname>`| |
+| delete | deletes specific CSAR Archives, Transformations or all of them, Usage: `delete <CSARname>`, `delete <Transformationname>` or `delete -c`/ `delete -t` / `delete -c -t` | `-c` or `--csar`, `-t` or `--transformation` |
+| list | shows all available CSAR Archives, Platforms & Transformations or specific Informations about them, Usage: `list`, `list -c` or `list <CSARname>` | `-c` or `--csar`, `-p` or `--platform`, `-t` or `--transformation` |
```

Comment:
What does `list <CSARname>` do? I think it returns information about a single csar file? But why to use list? list returns in my opinion multiple csars in this example.

Reply:
I think a command like `toscana csars list` better. 
Same for transformations `toscana transformations list`.
Deletion could be like this: `toscana transformation rm <transformation_name>`

Reply:
implemented it quite similar to your examples, will update docs with everything in it later today/ tomorrow

Reply:
Ok and please move the docs to a subfolder like `docs/dev/cli/`